crypto/cms/cms_sd.c

```diff
@@ -1026,8 +1036,11 @@ int CMS_SignerInfo_verify_content(CMS_SignerInfo *si, BIO *chain)
         if (EVP_PKEY_CTX_set_signature_md(pkctx, md) <= 0)
             goto err;
         si->pctx = pkctx;
-        if (!cms_sd_asn1_ctrl(si, 1))
+        if (!cms_sd_asn1_ctrl(si, 1)) {
```

Comment:
NOTE: Keep in mind that there is another recent PR related to cms_sd_asn1_ctrl() returning negative values currently..

Reply:
yes, that issue was resolved by making `cms_sd_asn1_ctrl()` return only 0/1